MetasysServices.Tests/MetasysClientTests.cs

```diff
@@ -2357,5 +2357,71 @@ public void TestMiscNullTokenValue()
         }
 
         #endregion
+
+        #region Ad-Hoc Calls
+        [Test]
+        public async Task TestSendAsyncWithAbsoluteUrl()
+        {
+            var requestUrl = "https://hostname/api/v5-preview/someController/someAction";
+            var httpRequest = new HttpRequestMessage(HttpMethod.Get, requestUrl);
+            await client.SendAsync(httpRequest);
+
+            httpTest.ShouldHaveCalled(requestUrl)
+                .WithVerb(HttpMethod.Get)
+                .Times(1);
+        }
+
+        [Test]
+        public async Task TestSendAsyncWithRelativeUrl()
+        {
+            var requestUrl = "api/v5-preview/someController/someAction";
```

Comment:
This relates to another comment I made. What rationale do you have for requiring consumers to include "`api/`?

Reply:
removed `api/` from request url.